cost-analyzer/templates/_helpers.tpl

```diff
@@ -467,18 +467,9 @@ Create the chart labels.
 {{- define "cost-analyzer.chartLabels" -}}
 helm.sh/chart: {{ include "cost-analyzer.chart" . }}
 app.kubernetes.io/managed-by: {{ .Release.Service }}
-{{- end -}}
-{{- define "kubecost.chartLabels" -}}
-app.kubernetes.io/name: {{ include "cost-analyzer.name" . }}
-helm.sh/chart: {{ include "cost-analyzer.chart" . }}
-app.kubernetes.io/instance: {{ .Release.Name }}
-app.kubernetes.io/managed-by: {{ .Release.Service }}
-{{- end -}}
-{{- define "kubecost.aggregator.chartLabels" -}}
-app.kubernetes.io/name: {{ include "aggregator.name" . }}
-helm.sh/chart: {{ include "cost-analyzer.chart" . }}
-app.kubernetes.io/instance: {{ .Release.Name }}
-app.kubernetes.io/managed-by: {{ .Release.Service }}
```

Comment:
Ishaan and I found that it was ok to delete `kubecost.chartLabels` and `kubecost.aggregator.chartLabels` because it is not referenced anywhere else in the Helm chart.